src/backend/api/meals.js

```diff
@@ -1,14 +1,79 @@
-const express = require("express");
+const express = require('express');
 const router = express.Router();
-const knex = require("../database");
+const knex = require('../database');
 
-router.get("/", async (request, response) => {
+router.get('/', async (req, res) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    const allMeals = await knex.select().table('meal');
+
+    if (allMeals.length !== 0) {
+      res.send(allMeals);
+    } else {
+      res.status(404).json({ error: 'Meals not found' });
```

Comment:
There is a possibility for `allMeals` to be none, so it won't be an error and hence the status won't be 404. Instead, the status can be 200 and send a message saying no meals found.